build.rs

```diff
@@ -534,7 +534,7 @@ fn configure(interpreter_version: &PythonVersion, lines: Vec<String>) -> Result<
     let libpath: &str = &lines[1];
     let enable_shared: &str = &lines[2];
     let ld_version: &str = &lines[3];
-    let exec_prefix: &str = &lines[4];
+    let base_prefix: &str = &lines[4];
```

Comment:
This type annotation(`: &str`) is redundant.
Could you please all `: &str` from these 4 lines?
They're not your fault, but it's nice timing to refactor them.